### DISCUSSION TITLE
JAK 1 @ 120fps how  ?

### DISCUSSION THREAD

**Discussion by drencoss** [open-goal/jak-project#3648] [Q&A]
My TV is @4k/120 the ingame options for JAK 1 only allow me to select 60/100/150 and the ALT options don't do anything if i change it there, how do i get this framerate ingame ?

**Comment by Hat-Kid** [comment]
You can't. The frame rate in Jak 1 has to be a divisor of 300. This is an engine limitation.

**Reply by drencoss** [reply]
> You can't. The frame rate in Jak 1 has to be a divisor of 300. This is an engine limitation.

Appreciate you letting me know Thx.
